project1/css/index.css

```diff
@@ -62,3 +62,102 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 10px;
 }
+
+nav {
+	padding-top:2%;
+	width:100%;
+	margin-bottom: 1px;
+	text-align:center;	
```

Comment:
Try to stay consistent with code style. Here you have `center` right up against the `:` but in previous lines you have a space between the `:` and value.